doc/HOW-TO/docker-build.rst

```diff
@@ -0,0 +1,29 @@
+Build a Qserv container image
+############################################################
+
+.. note::
+
+   This procedure is launched for each new commit inside `Travis-CI <https://travis-ci.org/github/lsst/qserv>`_.
+   It possible possible to retrieve the tag of the produced Qserv image on top of build log,
+   the log line start with `Building and testing qserv/qserv:<tag>`.
```

Comment:
Was this intended to be a link? This syntax is broken.

Reply:
This syntax works fine, please have a look at https://qserv.lsst.io/HOW-TO/docker-build.html?highlight=travis